packages/dart_frog_cli/lib/src/commands/update/update.dart

```diff
@@ -43,6 +54,13 @@ class UpdateCommand extends Command<int> {
     if (isUpToDate) {
       _logger.info('$packageName is already at the latest version.');
       return ExitCode.success.code;
+    } else if (verifyOnly) {
+      _logger
+        ..info('A new version of $packageName is available.\n')
+        ..info('The latest version: $latestVersion')
+        ..info('Your current version: $packageVersion\n')
+        ..info('To update, run "$executableName update".');
```

Comment:
**Nit:**

To mimic `flutter upgrade --verify-only`. I feel adding the word "now" adds a sense of urgency and avoids ambiguity.

```suggestion
        ..info('To update now, run "$executableName update".');
```